### ISSUE TITLE
Van Gogh's Mona Lisa is only 4.266 points???

### ISSUE THREAD

**Issue by h3clikejava** [titu1994/neural-image-assessment#64]
Use MobileNet Weights test Van Gogh's Mona Lisa only display NIMA Score : 4.266 +- (1.817)
Feels little practical

![image](https://user-images.githubusercontent.com/4688773/102440959-35e06780-405c-11eb-9afa-5b4afa3e66f1.png)

And there are another question about NASNet Mobile, I can get same result via MobileNet, but when I used NASNet will get different result in the same picture... Has anyone encountered the same problem? I have already and ensure resize the picture to 224x224

One more thing. This is my result by MobileNet
![image](https://user-images.githubusercontent.com/4688773/102468526-97b8c580-408c-11eb-8260-d70b6b5b3285.png)

**Comment by gohjiayi** [comment]
I agree that NIMA is rather practical, art is subjective even for humans. Interesting insights you have gathered!!

One possible high-level explanation I have for the difference in results - MobileNet and NASNet Mobile are light-weight models while NASNet is not. You can dig into the specifics if you're interested but it's commonly observed that they will behave differently (e.g. YOLO vs YOLO Tiny, although they are obj det models but you get the point!).
